shared/js/content-scripts/click-to-load.js

```diff
@@ -673,20 +673,17 @@
     })
 
     // Fetch reusable assets
-    sendMessage('getLoadingImage', 'light',
-        function (response) {
-            loadingImages.lightMode = response
-        })
+    sendMessage('getLoadingImage', 'light').then(response => {
```

Comment:
When looking at all this code before it all looks like it races, is that the case?

Can we file a follow up issue (not a priority) to consolidate these messages too as it's not needed anymore iiuc too.

Reply:
Yea, I think you're right that this code races. Agree that we should we should consolidate (or if possible just remove) these messages in the future.

Reply:
Can you file one or two follow up issues here?

Reply:
Done.